content/docs/cookbooks/tls-auth.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Securing Prometheus endpoints using TLS authentication
+sort_rank: 1
+---
+
+# Securing Prometheus endpoints using TLS auth
+
+Prometheus does *not* directly support authentication via [Transport Layer Security](https://en.wikipedia.org/wiki/Transport_Layer_Security), and this is so [by design](/docs/introduction/faq/#tls). In fact, Prometheus provides no server-side authentication, authorization, or encryption.
```

Comment:
TLS is encryption, not auth.

Reply:
In addition, Prometheus does support client cert authentication and basic auth. So this statement isn't exactly true.

Reply:
@SuperQ That may have been true at one point but is no longer the case: https://prometheus.io/docs/operating/security/#authentication/authorisation/encryption.

Reply:
That documentation pertains to the Prometheus UI itself, not what is available in the [scrape configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tls_config).

Prometheus itself is able to authenticate itself to targets.

I think the confusion here is that the section title is not specific enough. The use of "endpoints" is too vague, as this documentation is only about the Prometheus UI/API service, not targets.